website/docs/language/values/locals.mdx

```diff
@@ -12,8 +12,8 @@ description: >-
 > tutorial on HashiCorp Learn.
 
 A local value assigns a name to an [expression](/language/expressions),
-so you can use it multiple times within a module without repeating
-it.
+so you can use the name multiple times within a module without repeating
+the expression.
```

Comment:
Thank you - this is a good edit! I actually think we could even go a step further here - what do you think of this? 

```suggestion
so you can use the name multiple times within a module instead of repeating
the expression.
```